.github/workflows/integration-test.yml

```diff
@@ -117,10 +117,10 @@ jobs:
         with:
           go-version: ${{ env.GO_VERSION }}
           cache-dependency-path: '**/go.sum'
-      - name: Cache Docker images.
-        uses: AndreKurait/docker-cache@0fe76702a40db986d9663c24954fc14c6a6031b7 # 0.6.0
-        with:
-          key: docker-${{ matrix.ARCH }}
+      # - name: Cache Docker images.
```

Comment:
I think the fetch-depth change is more the cause for broken metrics but the confusing integration test failure has me commenting these out too.

Reply:
Let's just revert it for now

Reply:
https://github.com/signalfx/splunk-otel-collector/pull/6469

Reply:
let's just merge yours, that'll cover everything